/mnt/hwfile/huqinghao/miniconda3/bin/python
2024-05-17 07:36:59,693 - INFO - Note: detected 128 virtual cores but NumExpr set to maximum of 64, check "NUMEXPR_MAX_THREADS" environment variable.
2024-05-17 07:36:59,693 - INFO - Note: NumExpr detected 128 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.
2024-05-17 07:36:59,693 - INFO - NumExpr defaulting to 8 threads.
2024-05-17 07:37:00,802	INFO scripts.py:1182 -- Did not find any active Ray processes.

==========
== CUDA ==
==========

CUDA Version 12.1.0

Container image Copyright (c) 2016-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.

This container image and its contents are governed by the NVIDIA Deep Learning Container License.
By pulling and using the container, you accept the terms and conditions of this license:
https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license

A copy of this license is made available in this container at /NGC-DL-CONTAINER-LICENSE for your convenience.

*************************
** DEPRECATION NOTICE! **
*************************
THIS IMAGE IS DEPRECATED and is scheduled for DELETION.
    https://gitlab.com/nvidia/container-images/cuda/blob/master/doc/support-policy.md

Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380a00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381570>: Failed to establish a new connection: [Errno 111] Connection refused'))
INFO 05-17 07:37:15 api_server.py:191] vLLM API server version 0.3.3
INFO 05-17 07:37:15 api_server.py:192] args: Namespace(host=None, port=8000, uvicorn_log_level='info', allow_credentials=False, allowed_origins=['*'], allowed_methods=['*'], allowed_headers=['*'], api_key=None, served_model_name=None, lora_modules=[], delta_modules=[Delta(name='delta-1', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-1'), Delta(name='delta-2', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-2'), Delta(name='delta-3', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-3'), Delta(name='delta-4', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-4'), Delta(name='delta-5', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-5'), Delta(name='delta-6', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-6'), Delta(name='delta-7', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-7'), Delta(name='delta-8', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-8'), Delta(name='delta-9', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-9'), Delta(name='delta-10', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-10'), Delta(name='delta-11', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-11'), Delta(name='delta-12', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-12'), Delta(name='delta-13', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-13'), Delta(name='delta-14', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-14'), Delta(name='delta-15', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-15'), Delta(name='delta-16', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-16'), Delta(name='delta-17', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-17'), Delta(name='delta-18', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-18'), Delta(name='delta-19', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-19'), Delta(name='delta-20', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-20'), Delta(name='delta-21', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-21'), Delta(name='delta-22', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-22'), Delta(name='delta-23', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-23'), Delta(name='delta-24', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-24'), Delta(name='delta-25', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-25'), Delta(name='delta-26', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-26'), Delta(name='delta-27', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-27'), Delta(name='delta-28', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-28'), Delta(name='delta-29', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-29'), Delta(name='delta-30', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-30'), Delta(name='delta-31', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-31'), Delta(name='delta-32', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-32')], swap_modules=[], chat_template=None, response_role='assistant', ssl_keyfile=None, ssl_certfile=None, ssl_ca_certs=None, ssl_cert_reqs=0, root_path=None, middleware=[], model='/vllm/.idea/full_models/Llama-2-13b-hf', tokenizer=None, revision=None, code_revision=None, tokenizer_revision=None, tokenizer_mode='auto', trust_remote_code=False, download_dir=None, load_format='auto', dtype='auto', kv_cache_dtype='auto', max_model_len=None, worker_use_ray=False, pipeline_parallel_size=1, tensor_parallel_size=4, max_parallel_loading_workers=None, ray_workers_use_nsight=False, block_size=16, enable_prefix_caching=False, seed=0, swap_space=4, gpu_memory_utilization=0.85, max_num_batched_tokens=None, max_num_seqs=256, max_logprobs=5, disable_log_stats=False, quantization=None, enforce_eager=True, max_context_len_to_capture=8192, disable_custom_all_reduce=False, tokenizer_pool_size=0, tokenizer_pool_type='ray', tokenizer_pool_extra_config=None, enable_lora=False, max_loras=1, max_lora_rank=64, lora_extra_vocab_size=256, lora_dtype='auto', max_cpu_loras=32, enable_delta=True, max_deltas=32, max_cpu_deltas=32, max_delta_bitwidth=4, device='auto', image_input_type=None, image_token_id=None, image_input_shape=None, image_feature_size=None, scheduler_delay_factor=0.0, enable_prefetch=False, scheduler_policy='fcfs', max_swap_slots=1, max_cpu_models=4, enable_swap=False, engine_use_ray=False, disable_log_requests=True, max_log_len=None)
2024-05-17 07:37:19,564	INFO worker.py:1749 -- Started a local Ray instance.
INFO 05-17 07:37:21 llm_engine.py:90] Initializing an LLM engine (v0.3.3) with config: model='/vllm/.idea/full_models/Llama-2-13b-hf', tokenizer='/vllm/.idea/full_models/Llama-2-13b-hf', tokenizer_mode=auto, revision=None, tokenizer_revision=None, trust_remote_code=False, dtype=torch.float16, lora_config=None, delta_config=DeltaConfig(max_deltas=32, max_bitwidth=4, delta_dtype=torch.int8, max_cpu_deltas=32, delta_extra_vocab_size=0, pack_factor=Fraction(2, 1), kernel=<QuantKernel.TRITON: 'triton'>), swap_config=None, max_seq_len=4096, download_dir=None, load_format=auto, tensor_parallel_size=4, disable_custom_all_reduce=False, quantization=None, enforce_eager=True, kv_cache_dtype=auto, device_config=cuda, seed=0)
INFO 05-17 07:37:21 llm_engine.py:243] ignored verifying delta_config
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381e40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380f40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380670>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382710>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382fe0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f309435bd30>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382800>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381f30>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380f10>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943819f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943838b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381ea0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943814b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943805e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382170>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382a40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f309435bd30>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943823e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943803d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381420>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381c00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943835b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383580>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381bd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381270>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380dc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382590>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382e60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382290>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380580>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943816f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381a50>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943836a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383a60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381c60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381330>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380a00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382110>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f581c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380d60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381540>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943837c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383670>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f309435baf0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383dc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381ea0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383be0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381660>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380eb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382230>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943804c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381360>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943812d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381e40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f584f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382b00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381de0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381c00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Using bitblas
WARNING 05-17 07:46:56 models.py:48] Using bitblas delta modules
INFO 05-17 07:46:56 selector.py:16] Using FlashAttention backend.
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381420>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=98571)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7ecdb81ca170>
[36m(RayWorkerVllm pid=98571)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=98571)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=98571)[0m     self.remove()
[36m(RayWorkerVllm pid=98571)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=98571)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=98571)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=98949)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7ef26f9fa170>
[36m(RayWorkerVllm pid=98949)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=98949)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=98949)[0m     self.remove()
[36m(RayWorkerVllm pid=98949)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=98949)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=98949)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=98949)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7ef26f9fa170>
[36m(RayWorkerVllm pid=98949)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=98949)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=98949)[0m     self.remove()
[36m(RayWorkerVllm pid=98949)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=98949)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=98949)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380490>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f58ac0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380040>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943819c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381b70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943833d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f585e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383fa0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943832b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381e10>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381270>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943806a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f588b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381540>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381c90>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383d60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383eb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f58d00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383e20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381cc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381180>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943802b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380760>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f593c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380700>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=98571)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7ecdb81ca170>
[36m(RayWorkerVllm pid=98571)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=98571)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=98571)[0m     self.remove()
[36m(RayWorkerVllm pid=98571)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=98571)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=98571)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381240>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381db0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383760>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f58940>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943809d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383eb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383d60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381c90>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381660>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f59030>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381ff0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383c40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382fe0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382b30>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382200>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382c50>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381810>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383250>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383ca0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094380400>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f59690>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381ea0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383fa0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943836a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383fd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f309435bb50>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382320>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943805e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943834f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383b80>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943812a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f585e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943811e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381bd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383850>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383f70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f59750>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382320>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094382980>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381900>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943838b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381330>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f59cc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381ea0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383fa0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943836a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383fd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f59120>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381f30>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943827d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943822c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094381450>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3094383af0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=98571)[0m Using bitblas
[36m(RayWorkerVllm pid=98769)[0m Using bitblas
[36m(RayWorkerVllm pid=98949)[0m Using bitblas
[36m(RayWorkerVllm pid=98571)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=98571)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=98949)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=98949)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=98949)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=98949)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=98571)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=98571)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/837a1848ba0f066bae5b6c53a0bfb27ac232489344fb02d9df2a51efba2ff603/tvm_rt_mod'
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 07:55:46 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 07:55:46 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 07:55:53 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 07:55:53 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 07:59:52 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 07:59:52 selector.py:16] Using FlashAttention backend.
INFO 05-17 08:00:06 model_runner.py:161] Loading model weights took 6.1114 GB
INFO 05-17 08:00:06 worker_manager.py:297] Starting prefetching thread
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:00:07 model_runner.py:161] Loading model weights took 6.1114 GB
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:00:07 worker_manager.py:297] Starting prefetching thread
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:00:07 model_runner.py:161] Loading model weights took 6.1114 GB
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:00:07 worker_manager.py:297] Starting prefetching thread
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:00:07 model_runner.py:161] Loading model weights took 6.1114 GB
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:00:07 worker_manager.py:297] Starting prefetching thread
WARNING 05-17 08:00:09 quant_linear_triton.py:557] We should warm up the kernel here 
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3093f599c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f30943811e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:00:10 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:00:11 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:00:11 quant_linear_triton.py:557] We should warm up the kernel here 
INFO 05-17 08:00:24 ray_gpu_executor.py:275] # GPU blocks: 2362, # CPU blocks: 1310
INFO 05-17 08:00:27 scheduler.py:136] Using fcfs policy.
INFO 05-17 08:00:27 block_manager.py:264] disable automatic prefix caching
WARNING 05-17 08:00:27 serving_chat.py:373] No chat template provided. Chat API will not work.
INFO:     Started server process [90586]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     ::1:38406 - "GET /sysinfo HTTP/1.1" 200 OK
Translating from base-model to /vllm/.idea/full_models/Llama-2-13b-hf
Warming up starts
{'id': 3, 'prompt': 'USER: How to build an arena for chatbots?\nASSISTANT:', 'timestamp': 0, 'model': 'delta-11', 'min_tokens': 318, 'max_tokens': 318}
Prefetching is disabled
WARNING 05-17 08:00:32 worker_manager.py:344] Missed prefetch, fall back to loading delta 11 now
INFO 05-17 08:00:32 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:00:32 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:00:32 worker_manager.py:344] Missed prefetch, fall back to loading delta 11 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:00:32 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:00:32 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:00:32 worker_manager.py:344] Missed prefetch, fall back to loading delta 11 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:00:32 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:00:32 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:00:32 worker_manager.py:344] Missed prefetch, fall back to loading delta 11 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:00:32 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:00:32 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 08:00:34 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.720 seconds
INFO 05-17 08:00:34 worker_manager.py:364] Main thread loading delta 11 done
INFO 05-17 08:00:34 worker_manager.py:379] [1715904034.7057655] activating delta
INFO 05-17 08:00:34 worker_manager.py:384] [1715904034.8141832] CPU -> GPU time: 0.1084
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:00:35 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.189 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:00:35 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.221 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:00:35 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.221 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:00:35 worker_manager.py:364] Main thread loading delta 11 done
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:00:35 worker_manager.py:379] [1715904035.1676655] activating delta
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:00:35 worker_manager.py:364] Main thread loading delta 11 done
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:00:35 worker_manager.py:379] [1715904035.2005465] activating delta
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:00:35 worker_manager.py:364] Main thread loading delta 11 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:00:35 worker_manager.py:379] [1715904035.2007554] activating delta
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:00:35 worker_manager.py:384] [1715904035.2981257] CPU -> GPU time: 0.1304
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:00:35 worker_manager.py:384] [1715904035.3814569] CPU -> GPU time: 0.1809
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:00:35 worker_manager.py:384] [1715904035.3719163] CPU -> GPU time: 0.1711
INFO 05-17 08:00:35 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.1%, CPU KV cache usage: 0.0%
INFO 05-17 08:00:41 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.1%, CPU KV cache usage: 0.0%
INFO 05-17 08:00:46 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-17 08:00:51 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.3%, CPU KV cache usage: 0.0%
INFO 05-17 08:00:56 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.3%, CPU KV cache usage: 0.0%
INFO 05-17 08:01:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.3%, CPU KV cache usage: 0.0%
INFO 05-17 08:01:06 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.4%, CPU KV cache usage: 0.0%
INFO 05-17 08:01:11 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO 05-17 08:01:16 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO 05-17 08:01:21 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO 05-17 08:01:26 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO 05-17 08:01:31 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.7%, CPU KV cache usage: 0.0%
INFO 05-17 08:01:36 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.8%, CPU KV cache usage: 0.0%
INFO 05-17 08:01:41 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.8%, CPU KV cache usage: 0.0%
INFO 05-17 08:01:46 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.8%, CPU KV cache usage: 0.0%
INFO 05-17 08:01:51 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.9%, CPU KV cache usage: 0.0%
INFO:     ::1:38408 - "POST /v1/completions HTTP/1.1" 200 OK
Warming up ends
Issuing queries
sending 1 queries at 0.30000000000000004
sending 1 queries at 0.7000000000000001
sending 1 queries at 1.0
sending 1 queries at 1.3
sending 1 queries at 1.5
sending 1 queries at 1.8
sending 1 queries at 2.0
sending 1 queries at 2.8000000000000003
sending 1 queries at 3.9000000000000004
sending 1 queries at 4.0
sending 1 queries at 4.6000000000000005
sending 1 queries at 4.800000000000001
sending 1 queries at 5.1000000000000005
sending 4 queries at 6.0
sending 1 queries at 6.6000000000000005
sending 1 queries at 7.1000000000000005
sending 1 queries at 7.800000000000001
sending 1 queries at 9.1
sending 1 queries at 9.600000000000001
sending 1 queries at 9.8
sending 1 queries at 10.3
sending 1 queries at 10.4
sending 1 queries at 10.700000000000001
sending 1 queries at 10.8
sending 1 queries at 11.700000000000001
sending 1 queries at 12.0
sending 1 queries at 12.100000000000001
sending 1 queries at 12.200000000000001
sending 1 queries at 12.700000000000001
sending 1 queries at 12.9
sending 2 queries at 13.200000000000001
sending 1 queries at 13.600000000000001
sending 1 queries at 13.9
sending 1 queries at 14.200000000000001
sending 1 queries at 15.100000000000001
sending 1 queries at 15.5
sending 1 queries at 15.700000000000001
sending 1 queries at 15.9
sending 2 queries at 16.3
sending 1 queries at 16.7
sending 1 queries at 17.0
sending 2 queries at 17.1
sending 1 queries at 17.3
sending 1 queries at 17.400000000000002
sending 1 queries at 17.7
sending 1 queries at 17.900000000000002
sending 2 queries at 19.400000000000002
sending 1 queries at 19.5
sending 1 queries at 19.6
sending 1 queries at 19.900000000000002
sending 1 queries at 20.0
sending 1 queries at 20.200000000000003
sending 1 queries at 20.3
sending 2 queries at 20.400000000000002
sending 2 queries at 20.8
sending 1 queries at 20.900000000000002
sending 1 queries at 21.0
sending 2 queries at 21.6
sending 2 queries at 22.3
sending 1 queries at 23.5
sending 1 queries at 23.700000000000003
sending 1 queries at 25.0
sending 1 queries at 25.3
sending 2 queries at 25.8
sending 2 queries at 25.900000000000002
sending 1 queries at 26.0
sending 1 queries at 26.1
sending 1 queries at 26.200000000000003
sending 2 queries at 26.400000000000002
sending 1 queries at 26.8
sending 1 queries at 27.1
sending 1 queries at 27.200000000000003
sending 1 queries at 27.400000000000002
sending 1 queries at 27.5
sending 1 queries at 27.8
sending 1 queries at 28.6
sending 1 queries at 28.8
sending 1 queries at 29.1
sending 1 queries at 29.200000000000003
sending 1 queries at 29.6
sending 1 queries at 29.700000000000003
sending 1 queries at 29.8
Prefetching is disabled
Prefetching is disabled
WARNING 05-17 08:01:52 worker_manager.py:344] Missed prefetch, fall back to loading delta 22 now
INFO 05-17 08:01:52 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:01:52 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:01:52 worker_manager.py:344] Missed prefetch, fall back to loading delta 22 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:01:52 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:01:52 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:01:52 worker_manager.py:344] Missed prefetch, fall back to loading delta 22 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:01:52 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:01:52 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:01:52 worker_manager.py:344] Missed prefetch, fall back to loading delta 22 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:01:52 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:01:52 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-17 08:01:54 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.707 seconds
INFO 05-17 08:01:54 worker_manager.py:364] Main thread loading delta 22 done
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:01:55 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.169 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:01:55 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.185 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:01:55 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.177 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:01:55 worker_manager.py:364] Main thread loading delta 22 done
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:01:55 worker_manager.py:364] Main thread loading delta 22 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:01:55 worker_manager.py:364] Main thread loading delta 22 done
Prefetching is disabled
WARNING 05-17 08:01:55 worker_manager.py:344] Missed prefetch, fall back to loading delta 1 now
INFO 05-17 08:01:55 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:01:55 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:01:55 worker_manager.py:344] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:01:55 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:01:55 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:01:55 worker_manager.py:344] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:01:55 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:01:55 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:01:55 worker_manager.py:344] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:01:55 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:01:55 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-17 08:01:57 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.692 seconds
INFO 05-17 08:01:57 worker_manager.py:364] Main thread loading delta 1 done
WARNING 05-17 08:01:57 worker_manager.py:344] Missed prefetch, fall back to loading delta 2 now
INFO 05-17 08:01:57 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:01:57 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 08:01:57 metrics.py:276] Avg prompt throughput: 7.5 tokens/s, Avg generation throughput: 1.4 tokens/s, Running: 8 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:01:57 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.241 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:01:57 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.205 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:01:57 worker_manager.py:364] Main thread loading delta 1 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:01:57 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.242 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:01:57 worker_manager.py:364] Main thread loading delta 1 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:01:57 worker_manager.py:364] Main thread loading delta 1 done
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:01:58 worker_manager.py:344] Missed prefetch, fall back to loading delta 2 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:01:58 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:01:58 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:01:58 worker_manager.py:344] Missed prefetch, fall back to loading delta 2 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:01:58 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:01:58 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:01:58 worker_manager.py:344] Missed prefetch, fall back to loading delta 2 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:01:58 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:01:58 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
INFO 05-17 08:01:59 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.612 seconds
INFO 05-17 08:01:59 worker_manager.py:364] Main thread loading delta 2 done
Prefetching is disabled
WARNING 05-17 08:01:59 worker_manager.py:344] Missed prefetch, fall back to loading delta 6 now
INFO 05-17 08:01:59 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:01:59 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:00 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.180 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:00 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.154 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:00 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.157 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:00 worker_manager.py:364] Main thread loading delta 2 done
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:00 worker_manager.py:364] Main thread loading delta 2 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:00 worker_manager.py:364] Main thread loading delta 2 done
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:00 worker_manager.py:344] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:00 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:00 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:00 worker_manager.py:344] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:00 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:00 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:00 worker_manager.py:344] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:00 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:00 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 08:02:00 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.602 seconds
INFO 05-17 08:02:00 worker_manager.py:364] Main thread loading delta 6 done
WARNING 05-17 08:02:01 worker_manager.py:344] Missed prefetch, fall back to loading delta 10 now
INFO 05-17 08:02:01 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:01 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:02 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.226 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:02 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.228 seconds
INFO 05-17 08:02:02 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.605 seconds
INFO 05-17 08:02:02 worker_manager.py:364] Main thread loading delta 10 done
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:02 worker_manager.py:364] Main thread loading delta 6 done
Prefetching is disabled
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:02 worker_manager.py:364] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:02 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.217 seconds
Prefetching is disabled
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:02 worker_manager.py:344] Missed prefetch, fall back to loading delta 10 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:02 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:02 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:02 worker_manager.py:364] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:02 worker_manager.py:344] Missed prefetch, fall back to loading delta 10 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:02 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:02 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:02 worker_manager.py:344] Missed prefetch, fall back to loading delta 10 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:02 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:02 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:04 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.965 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:04 worker_manager.py:364] Main thread loading delta 10 done
Prefetching is disabled
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:04 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.007 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:04 worker_manager.py:364] Main thread loading delta 10 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:04 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.969 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:04 worker_manager.py:364] Main thread loading delta 10 done
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-17 08:02:06 metrics.py:276] Avg prompt throughput: 13.3 tokens/s, Avg generation throughput: 0.7 tokens/s, Running: 8 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
WARNING 05-17 08:02:06 worker_manager.py:344] Missed prefetch, fall back to loading delta 3 now
INFO 05-17 08:02:06 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:06 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:06 worker_manager.py:344] Missed prefetch, fall back to loading delta 3 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:06 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:06 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:06 worker_manager.py:344] Missed prefetch, fall back to loading delta 3 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:06 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:06 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:06 worker_manager.py:344] Missed prefetch, fall back to loading delta 3 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:06 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:06 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-17 08:02:07 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.643 seconds
INFO 05-17 08:02:07 worker_manager.py:364] Main thread loading delta 3 done
Prefetching is disabled
WARNING 05-17 08:02:08 worker_manager.py:344] Missed prefetch, fall back to loading delta 4 now
INFO 05-17 08:02:08 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:08 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:08 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.211 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:08 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.234 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:08 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.235 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:08 worker_manager.py:364] Main thread loading delta 3 done
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:08 worker_manager.py:364] Main thread loading delta 3 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:08 worker_manager.py:364] Main thread loading delta 3 done
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:08 worker_manager.py:344] Missed prefetch, fall back to loading delta 4 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:08 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:08 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:08 worker_manager.py:344] Missed prefetch, fall back to loading delta 4 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:08 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:08 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:08 worker_manager.py:344] Missed prefetch, fall back to loading delta 4 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:08 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:08 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-17 08:02:09 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.570 seconds
INFO 05-17 08:02:09 worker_manager.py:364] Main thread loading delta 4 done
WARNING 05-17 08:02:09 worker_manager.py:344] Missed prefetch, fall back to loading delta 5 now
Prefetching is disabled
INFO 05-17 08:02:09 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:09 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:10 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.107 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:10 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.085 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:10 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.097 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:10 worker_manager.py:364] Main thread loading delta 4 done
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:10 worker_manager.py:364] Main thread loading delta 4 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:10 worker_manager.py:364] Main thread loading delta 4 done
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:10 worker_manager.py:344] Missed prefetch, fall back to loading delta 5 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:10 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:10 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:10 worker_manager.py:344] Missed prefetch, fall back to loading delta 5 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:10 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:10 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:10 worker_manager.py:344] Missed prefetch, fall back to loading delta 5 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:10 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:10 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 08:02:11 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.634 seconds
INFO 05-17 08:02:11 worker_manager.py:364] Main thread loading delta 5 done
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
WARNING 05-17 08:02:11 worker_manager.py:344] Missed prefetch, fall back to loading delta 7 now
INFO 05-17 08:02:11 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:11 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:13 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.220 seconds
INFO 05-17 08:02:13 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.589 seconds
INFO 05-17 08:02:13 worker_manager.py:364] Main thread loading delta 7 done
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:13 worker_manager.py:364] Main thread loading delta 5 done
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:13 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.315 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:13 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.268 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:13 worker_manager.py:364] Main thread loading delta 5 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:13 worker_manager.py:364] Main thread loading delta 5 done
WARNING 05-17 08:02:13 worker_manager.py:344] Missed prefetch, fall back to loading delta 9 now
INFO 05-17 08:02:13 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:13 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:13 worker_manager.py:344] Missed prefetch, fall back to loading delta 7 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:13 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:13 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:13 worker_manager.py:344] Missed prefetch, fall back to loading delta 7 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:13 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:13 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:13 worker_manager.py:344] Missed prefetch, fall back to loading delta 7 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:13 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:13 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-17 08:02:15 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.594 seconds
INFO 05-17 08:02:15 worker_manager.py:364] Main thread loading delta 9 done
WARNING 05-17 08:02:15 worker_manager.py:344] Missed prefetch, fall back to loading delta 13 now
INFO 05-17 08:02:15 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:15 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:15 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.116 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:15 worker_manager.py:364] Main thread loading delta 7 done
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:15 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.106 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:15 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.096 seconds
Prefetching is disabled
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:15 worker_manager.py:344] Missed prefetch, fall back to loading delta 9 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:15 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:15 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:15 worker_manager.py:364] Main thread loading delta 7 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:15 worker_manager.py:364] Main thread loading delta 7 done
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:15 worker_manager.py:344] Missed prefetch, fall back to loading delta 9 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:15 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:15 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:15 worker_manager.py:344] Missed prefetch, fall back to loading delta 9 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:15 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:15 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 08:02:16 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.632 seconds
INFO 05-17 08:02:16 worker_manager.py:364] Main thread loading delta 13 done
WARNING 05-17 08:02:17 worker_manager.py:344] Missed prefetch, fall back to loading delta 14 now
INFO 05-17 08:02:17 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:17 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-17 08:02:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 37 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.5%, CPU KV cache usage: 0.0%
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:17 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.082 seconds
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:17 worker_manager.py:364] Main thread loading delta 9 done
Prefetching is disabled
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:18 worker_manager.py:344] Missed prefetch, fall back to loading delta 13 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:18 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:18 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:18 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.085 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:18 worker_manager.py:364] Main thread loading delta 9 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:18 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.151 seconds
Prefetching is disabled
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:18 worker_manager.py:364] Main thread loading delta 9 done
Prefetching is disabled
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:18 worker_manager.py:344] Missed prefetch, fall back to loading delta 13 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:18 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:18 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:18 worker_manager.py:344] Missed prefetch, fall back to loading delta 13 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:18 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:18 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
INFO 05-17 08:02:18 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.584 seconds
INFO 05-17 08:02:18 worker_manager.py:364] Main thread loading delta 14 done
WARNING 05-17 08:02:18 worker_manager.py:344] Missed prefetch, fall back to loading delta 15 now
INFO 05-17 08:02:18 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:18 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:20 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.143 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:20 worker_manager.py:364] Main thread loading delta 13 done
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:20 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.087 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:20 worker_manager.py:364] Main thread loading delta 13 done
INFO 05-17 08:02:20 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.684 seconds
INFO 05-17 08:02:20 worker_manager.py:364] Main thread loading delta 15 done
total threads: 96
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:20 worker_manager.py:344] Missed prefetch, fall back to loading delta 14 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:20 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:20 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:20 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.134 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:20 worker_manager.py:364] Main thread loading delta 13 done
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:20 worker_manager.py:344] Missed prefetch, fall back to loading delta 14 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:20 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:20 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:20 worker_manager.py:344] Missed prefetch, fall back to loading delta 14 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:20 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:20 models.py:263] [main] Lossless Compression Disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
Prefetching is disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:22 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.991 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:22 worker_manager.py:364] Main thread loading delta 14 done
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:22 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.010 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:22 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.957 seconds
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:22 worker_manager.py:344] Missed prefetch, fall back to loading delta 15 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:22 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:22 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:22 worker_manager.py:364] Main thread loading delta 14 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:22 worker_manager.py:364] Main thread loading delta 14 done
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:22 worker_manager.py:344] Missed prefetch, fall back to loading delta 15 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:22 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:22 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:22 worker_manager.py:344] Missed prefetch, fall back to loading delta 15 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:22 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:22 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:24 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.936 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:24 worker_manager.py:364] Main thread loading delta 15 done
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:24 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.947 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:24 worker_manager.py:364] Main thread loading delta 15 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:24 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.926 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:24 worker_manager.py:364] Main thread loading delta 15 done
INFO 05-17 08:02:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 37 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.5%, CPU KV cache usage: 0.0%
INFO:     ::1:38640 - "POST /v1/completions HTTP/1.1" 200 OK
WARNING 05-17 08:02:28 worker_manager.py:344] Missed prefetch, fall back to loading delta 32 now
INFO 05-17 08:02:28 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:28 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:28 worker_manager.py:344] Missed prefetch, fall back to loading delta 32 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:28 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:28 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:28 worker_manager.py:344] Missed prefetch, fall back to loading delta 32 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:28 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:28 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:28 worker_manager.py:344] Missed prefetch, fall back to loading delta 32 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:28 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:28 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 08:02:30 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.576 seconds
INFO 05-17 08:02:30 worker_manager.py:364] Main thread loading delta 32 done
WARNING 05-17 08:02:30 worker_manager.py:344] Missed prefetch, fall back to loading delta 12 now
INFO 05-17 08:02:30 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:30 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:30 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.127 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:30 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.131 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:30 worker_manager.py:364] Main thread loading delta 32 done
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:30 worker_manager.py:364] Main thread loading delta 32 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:30 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.188 seconds
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:30 worker_manager.py:344] Missed prefetch, fall back to loading delta 12 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:30 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:30 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:30 worker_manager.py:364] Main thread loading delta 32 done
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:30 worker_manager.py:344] Missed prefetch, fall back to loading delta 12 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:30 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:30 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:30 worker_manager.py:344] Missed prefetch, fall back to loading delta 12 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:30 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:30 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 08:02:31 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.617 seconds
INFO 05-17 08:02:32 worker_manager.py:364] Main thread loading delta 12 done
WARNING 05-17 08:02:32 worker_manager.py:344] Missed prefetch, fall back to loading delta 17 now
INFO 05-17 08:02:32 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:32 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:32 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.091 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:33 worker_manager.py:364] Main thread loading delta 12 done
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:33 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.145 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:33 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.086 seconds
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:33 worker_manager.py:344] Missed prefetch, fall back to loading delta 17 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:33 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:33 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:33 worker_manager.py:364] Main thread loading delta 12 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:33 worker_manager.py:364] Main thread loading delta 12 done
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:33 worker_manager.py:344] Missed prefetch, fall back to loading delta 17 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:33 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:33 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:33 worker_manager.py:344] Missed prefetch, fall back to loading delta 17 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:33 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:33 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 08:02:33 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.572 seconds
INFO 05-17 08:02:33 worker_manager.py:364] Main thread loading delta 17 done
WARNING 05-17 08:02:33 worker_manager.py:344] Missed prefetch, fall back to loading delta 18 now
INFO 05-17 08:02:33 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:33 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:35 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.105 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:35 worker_manager.py:364] Main thread loading delta 17 done
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:35 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.113 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:35 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.106 seconds
INFO 05-17 08:02:35 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.637 seconds
INFO 05-17 08:02:35 worker_manager.py:364] Main thread loading delta 18 done
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:35 worker_manager.py:344] Missed prefetch, fall back to loading delta 18 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:35 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:35 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:35 worker_manager.py:364] Main thread loading delta 17 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:35 worker_manager.py:364] Main thread loading delta 17 done
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:35 worker_manager.py:344] Missed prefetch, fall back to loading delta 18 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:35 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:35 models.py:263] [main] Lossless Compression Disabled
WARNING 05-17 08:02:35 worker_manager.py:344] Missed prefetch, fall back to loading delta 19 now
INFO 05-17 08:02:35 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:35 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:35 worker_manager.py:344] Missed prefetch, fall back to loading delta 18 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:35 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:35 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 08:02:37 metrics.py:276] Avg prompt throughput: 84.9 tokens/s, Avg generation throughput: 2.9 tokens/s, Running: 95 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 10.4%, CPU KV cache usage: 0.0%
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:37 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.093 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:37 worker_manager.py:364] Main thread loading delta 18 done
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:37 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.082 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:37 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.033 seconds
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:37 worker_manager.py:344] Missed prefetch, fall back to loading delta 19 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:37 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:37 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:37 worker_manager.py:364] Main thread loading delta 18 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:37 worker_manager.py:364] Main thread loading delta 18 done
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:38 worker_manager.py:344] Missed prefetch, fall back to loading delta 19 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:38 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:38 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:02:38 worker_manager.py:344] Missed prefetch, fall back to loading delta 19 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:38 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:38 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 08:02:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 95 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 10.4%, CPU KV cache usage: 0.0%
INFO 05-17 08:02:53 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 18.112 seconds
INFO 05-17 08:02:54 worker_manager.py:364] Main thread loading delta 19 done
WARNING 05-17 08:02:54 worker_manager.py:344] Missed prefetch, fall back to loading delta 20 now
INFO 05-17 08:02:54 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:54 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 08:02:56 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.615 seconds
INFO 05-17 08:02:56 worker_manager.py:364] Main thread loading delta 20 done
WARNING 05-17 08:02:56 worker_manager.py:344] Missed prefetch, fall back to loading delta 26 now
INFO 05-17 08:02:56 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 08:02:56 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:57 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 19.358 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:57 worker_manager.py:364] Main thread loading delta 19 done
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:02:57 worker_manager.py:344] Missed prefetch, fall back to loading delta 20 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:57 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:57 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 08:02:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 95 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 10.4%, CPU KV cache usage: 0.0%
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:57 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 19.547 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:57 worker_manager.py:364] Main thread loading delta 19 done
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:02:57 worker_manager.py:344] Missed prefetch, fall back to loading delta 20 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:57 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:02:57 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 08:02:58 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.785 seconds
INFO 05-17 08:02:58 worker_manager.py:364] Main thread loading delta 26 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:59 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 21.847 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:02:59 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.494 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:02:59 worker_manager.py:364] Main thread loading delta 19 done
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:03:00 worker_manager.py:364] Main thread loading delta 20 done
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:03:00 worker_manager.py:344] Missed prefetch, fall back to loading delta 20 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:03:00 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:03:00 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m WARNING 05-17 08:03:00 worker_manager.py:344] Missed prefetch, fall back to loading delta 26 now
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:03:00 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:03:00 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:03:00 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.626 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:03:00 worker_manager.py:364] Main thread loading delta 20 done
[36m(RayWorkerVllm pid=98769)[0m WARNING 05-17 08:03:00 worker_manager.py:344] Missed prefetch, fall back to loading delta 26 now
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:03:00 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:03:00 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:03:02 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.400 seconds
[36m(RayWorkerVllm pid=98571)[0m INFO 05-17 08:03:02 worker_manager.py:364] Main thread loading delta 26 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:03:02 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.661 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:03:02 worker_manager.py:364] Main thread loading delta 20 done
[36m(RayWorkerVllm pid=98949)[0m WARNING 05-17 08:03:02 worker_manager.py:344] Missed prefetch, fall back to loading delta 26 now
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:03:02 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:03:02 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:03:03 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.504 seconds
[36m(RayWorkerVllm pid=98769)[0m INFO 05-17 08:03:03 worker_manager.py:364] Main thread loading delta 26 done
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:03:05 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.318 seconds
[36m(RayWorkerVllm pid=98949)[0m INFO 05-17 08:03:05 worker_manager.py:364] Main thread loading delta 26 done
INFO 05-17 08:03:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 95 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 10.4%, CPU KV cache usage: 0.0%
INFO 05-17 08:03:14 metrics.py:276] Avg prompt throughput: 326.7 tokens/s, Avg generation throughput: 23.5 tokens/s, Running: 94 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 10.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38682 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:03:20 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 31.4 tokens/s, Running: 94 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 11.0%, CPU KV cache usage: 0.0%
INFO:     ::1:38630 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38674 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:03:26 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 30.9 tokens/s, Running: 88 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 10.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38692 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38694 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38834 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38700 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38686 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:03:31 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 30.2 tokens/s, Running: 87 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 10.9%, CPU KV cache usage: 0.0%
INFO:     ::1:38806 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38810 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38716 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:03:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 15.4 tokens/s, Running: 84 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 11.0%, CPU KV cache usage: 0.0%
INFO:     ::1:38698 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:03:43 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 42.9 tokens/s, Running: 82 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 11.2%, CPU KV cache usage: 0.0%
INFO:     ::1:38626 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:03:49 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 28.7 tokens/s, Running: 82 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 11.9%, CPU KV cache usage: 0.0%
INFO 05-17 08:03:54 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 28.7 tokens/s, Running: 82 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 12.5%, CPU KV cache usage: 0.0%
INFO 05-17 08:04:00 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 28.6 tokens/s, Running: 82 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 13.2%, CPU KV cache usage: 0.0%
INFO 05-17 08:04:06 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 28.5 tokens/s, Running: 82 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 13.4%, CPU KV cache usage: 0.0%
INFO 05-17 08:04:12 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 28.6 tokens/s, Running: 82 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 13.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38844 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:04:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 14.9 tokens/s, Running: 81 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 13.8%, CPU KV cache usage: 0.0%
INFO:     ::1:38728 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:04:23 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 39.8 tokens/s, Running: 80 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.0%, CPU KV cache usage: 0.0%
INFO 05-17 08:04:29 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 28.0 tokens/s, Running: 80 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.4%, CPU KV cache usage: 0.0%
INFO:     ::1:38718 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:04:34 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 28.5 tokens/s, Running: 79 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.9%, CPU KV cache usage: 0.0%
INFO:     ::1:38804 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:04:40 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 28.9 tokens/s, Running: 77 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 15.1%, CPU KV cache usage: 0.0%
INFO:     ::1:38668 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:04:45 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 27.8 tokens/s, Running: 77 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 15.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38830 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38638 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:04:51 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 27.5 tokens/s, Running: 75 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 15.6%, CPU KV cache usage: 0.0%
INFO 05-17 08:04:56 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 27.3 tokens/s, Running: 75 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 15.9%, CPU KV cache usage: 0.0%
INFO 05-17 08:05:02 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 27.3 tokens/s, Running: 75 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 16.1%, CPU KV cache usage: 0.0%
INFO:     ::1:38736 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:05:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 14.6 tokens/s, Running: 74 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 16.3%, CPU KV cache usage: 0.0%
INFO:     ::1:38818 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:05:13 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 37.5 tokens/s, Running: 72 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 16.2%, CPU KV cache usage: 0.0%
INFO:     ::1:38622 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:05:19 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 25.7 tokens/s, Running: 71 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 16.6%, CPU KV cache usage: 0.0%
INFO:     ::1:38664 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38758 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:05:24 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 25.8 tokens/s, Running: 70 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 17.0%, CPU KV cache usage: 0.0%
INFO:     ::1:38752 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:05:30 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 25.1 tokens/s, Running: 68 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 16.9%, CPU KV cache usage: 0.0%
INFO:     ::1:38838 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:05:35 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 24.6 tokens/s, Running: 68 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 17.1%, CPU KV cache usage: 0.0%
INFO 05-17 08:05:40 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 26.1 tokens/s, Running: 68 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 17.3%, CPU KV cache usage: 0.0%
INFO 05-17 08:05:46 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 26.2 tokens/s, Running: 68 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 17.5%, CPU KV cache usage: 0.0%
INFO 05-17 08:05:51 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 26.1 tokens/s, Running: 68 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 17.8%, CPU KV cache usage: 0.0%
INFO 05-17 08:05:56 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 26.1 tokens/s, Running: 68 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 18.1%, CPU KV cache usage: 0.0%
INFO:     ::1:38608 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38780 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:06:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 26.0 tokens/s, Running: 66 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 18.3%, CPU KV cache usage: 0.0%
INFO 05-17 08:06:06 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 25.3 tokens/s, Running: 66 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 18.8%, CPU KV cache usage: 0.0%
INFO 05-17 08:06:11 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 25.4 tokens/s, Running: 64 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 18.8%, CPU KV cache usage: 0.0%
INFO:     ::1:38756 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38720 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38610 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:06:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.8 tokens/s, Running: 63 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 18.8%, CPU KV cache usage: 0.0%
INFO 05-17 08:06:24 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 27.8 tokens/s, Running: 63 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 19.0%, CPU KV cache usage: 0.0%
INFO:     ::1:38730 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:06:31 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 25.6 tokens/s, Running: 62 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 19.1%, CPU KV cache usage: 0.0%
INFO 05-17 08:06:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 21.0 tokens/s, Running: 62 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 19.6%, CPU KV cache usage: 0.0%
INFO 05-17 08:06:43 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 29.4 tokens/s, Running: 61 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 19.8%, CPU KV cache usage: 0.0%
INFO:     ::1:38820 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38816 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38848 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:06:51 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 25.0 tokens/s, Running: 59 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 20.0%, CPU KV cache usage: 0.0%
INFO:     ::1:38634 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38774 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:06:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 18.4 tokens/s, Running: 57 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 19.4%, CPU KV cache usage: 0.0%
INFO 05-17 08:07:03 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 30.4 tokens/s, Running: 57 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 19.6%, CPU KV cache usage: 0.0%
INFO:     ::1:38864 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:07:10 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 23.4 tokens/s, Running: 56 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 19.7%, CPU KV cache usage: 0.0%
INFO 05-17 08:07:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 15.6 tokens/s, Running: 56 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 20.2%, CPU KV cache usage: 0.0%
INFO 05-17 08:07:24 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 30.4 tokens/s, Running: 55 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 20.6%, CPU KV cache usage: 0.0%
INFO:     ::1:38654 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:07:32 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.6 tokens/s, Running: 55 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 21.1%, CPU KV cache usage: 0.0%
INFO:     ::1:38652 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38740 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:07:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 20.6 tokens/s, Running: 53 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 20.5%, CPU KV cache usage: 0.0%
INFO:     ::1:38704 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38612 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:07:44 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 24.0 tokens/s, Running: 51 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 20.0%, CPU KV cache usage: 0.0%
INFO:     ::1:38636 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:07:50 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.0 tokens/s, Running: 50 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 20.0%, CPU KV cache usage: 0.0%
INFO 05-17 08:07:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 15.0 tokens/s, Running: 50 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 20.6%, CPU KV cache usage: 0.0%
INFO 05-17 08:08:04 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 27.8 tokens/s, Running: 50 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 21.1%, CPU KV cache usage: 0.0%
INFO:     ::1:38628 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:08:11 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.2 tokens/s, Running: 49 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 21.0%, CPU KV cache usage: 0.0%
INFO 05-17 08:08:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 16.0 tokens/s, Running: 49 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 21.3%, CPU KV cache usage: 0.0%
INFO 05-17 08:08:24 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 29.0 tokens/s, Running: 49 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 21.7%, CPU KV cache usage: 0.0%
INFO 05-17 08:08:30 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.9 tokens/s, Running: 49 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 22.1%, CPU KV cache usage: 0.0%
INFO:     ::1:38764 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38726 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:08:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.9 tokens/s, Running: 47 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 21.8%, CPU KV cache usage: 0.0%
INFO 05-17 08:08:43 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 23.4 tokens/s, Running: 47 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 22.3%, CPU KV cache usage: 0.0%
INFO 05-17 08:08:49 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 23.0 tokens/s, Running: 47 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 22.4%, CPU KV cache usage: 0.0%
INFO 05-17 08:08:55 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 23.3 tokens/s, Running: 47 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 22.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38602 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:09:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 23.7 tokens/s, Running: 46 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 22.7%, CPU KV cache usage: 0.0%
INFO 05-17 08:09:06 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 24.4 tokens/s, Running: 46 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.1%, CPU KV cache usage: 0.0%
INFO 05-17 08:09:12 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 24.5 tokens/s, Running: 46 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.7%, CPU KV cache usage: 0.0%
INFO 05-17 08:09:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 17.9 tokens/s, Running: 46 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.9%, CPU KV cache usage: 0.0%
INFO:     ::1:38680 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:09:23 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 29.7 tokens/s, Running: 45 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.7%, CPU KV cache usage: 0.0%
INFO 05-17 08:09:29 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 23.9 tokens/s, Running: 44 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.5%, CPU KV cache usage: 0.0%
INFO:     ::1:38734 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38670 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:09:35 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 23.0 tokens/s, Running: 43 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.5%, CPU KV cache usage: 0.0%
INFO 05-17 08:09:40 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 23.1 tokens/s, Running: 43 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 24.0%, CPU KV cache usage: 0.0%
INFO 05-17 08:09:46 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 23.1 tokens/s, Running: 43 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 24.2%, CPU KV cache usage: 0.0%
INFO:     ::1:38600 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:09:51 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.7 tokens/s, Running: 42 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.9%, CPU KV cache usage: 0.0%
INFO:     ::1:38776 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:09:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.4 tokens/s, Running: 41 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.7%, CPU KV cache usage: 0.0%
INFO 05-17 08:10:03 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.1 tokens/s, Running: 41 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 24.0%, CPU KV cache usage: 0.0%
INFO 05-17 08:10:08 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.0 tokens/s, Running: 40 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 24.0%, CPU KV cache usage: 0.0%
INFO:     ::1:38710 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38784 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:10:15 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 23.9 tokens/s, Running: 39 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.8%, CPU KV cache usage: 0.0%
INFO:     ::1:38746 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:10:21 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 24.2 tokens/s, Running: 37 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 22.9%, CPU KV cache usage: 0.0%
INFO:     ::1:38666 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:10:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 18.5 tokens/s, Running: 37 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.3%, CPU KV cache usage: 0.0%
INFO 05-17 08:10:32 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 28.5 tokens/s, Running: 37 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38738 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:10:39 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.7 tokens/s, Running: 36 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.6%, CPU KV cache usage: 0.0%
INFO 05-17 08:10:45 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.5 tokens/s, Running: 36 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.8%, CPU KV cache usage: 0.0%
INFO:     ::1:38762 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:10:51 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.4 tokens/s, Running: 35 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.5%, CPU KV cache usage: 0.0%
INFO:     ::1:38826 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:10:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 18.5 tokens/s, Running: 34 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.3%, CPU KV cache usage: 0.0%
INFO 05-17 08:11:03 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 24.5 tokens/s, Running: 34 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38604 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38782 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:11:09 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 20.7 tokens/s, Running: 32 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 22.6%, CPU KV cache usage: 0.0%
INFO 05-17 08:11:15 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 20.2 tokens/s, Running: 32 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 22.9%, CPU KV cache usage: 0.0%
INFO 05-17 08:11:22 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 20.1 tokens/s, Running: 32 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 23.3%, CPU KV cache usage: 0.0%
INFO:     ::1:38812 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38760 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:11:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 17.9 tokens/s, Running: 30 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 21.5%, CPU KV cache usage: 0.0%
INFO:     ::1:38866 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:11:33 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 20.7 tokens/s, Running: 29 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 21.0%, CPU KV cache usage: 0.0%
INFO:     ::1:38794 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38742 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:11:39 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 17.7 tokens/s, Running: 27 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 19.6%, CPU KV cache usage: 0.0%
INFO 05-17 08:11:45 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 17.2 tokens/s, Running: 27 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 20.0%, CPU KV cache usage: 0.0%
INFO:     ::1:38732 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38786 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:11:52 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 16.5 tokens/s, Running: 24 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 17.4%, CPU KV cache usage: 0.0%
INFO:     ::1:38796 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38750 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38772 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:11:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.7 tokens/s, Running: 22 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 16.0%, CPU KV cache usage: 0.0%
INFO:     ::1:38840 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38822 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:12:02 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 16.2 tokens/s, Running: 20 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.7%, CPU KV cache usage: 0.0%
INFO 05-17 08:12:08 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 14.0 tokens/s, Running: 20 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.9%, CPU KV cache usage: 0.0%
INFO 05-17 08:12:14 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 14.1 tokens/s, Running: 20 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 15.2%, CPU KV cache usage: 0.0%
INFO:     ::1:38618 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:12:19 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 13.7 tokens/s, Running: 19 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.6%, CPU KV cache usage: 0.0%
INFO 05-17 08:12:25 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 13.3 tokens/s, Running: 19 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.8%, CPU KV cache usage: 0.0%
INFO 05-17 08:12:31 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 13.4 tokens/s, Running: 18 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.2%, CPU KV cache usage: 0.0%
INFO:     ::1:38672 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:12:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.6 tokens/s, Running: 18 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.5%, CPU KV cache usage: 0.0%
INFO 05-17 08:12:42 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.7 tokens/s, Running: 18 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.6%, CPU KV cache usage: 0.0%
INFO 05-17 08:12:48 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.6 tokens/s, Running: 18 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.8%, CPU KV cache usage: 0.0%
INFO 05-17 08:12:54 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.7 tokens/s, Running: 18 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.9%, CPU KV cache usage: 0.0%
INFO 05-17 08:12:59 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.6 tokens/s, Running: 18 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 15.3%, CPU KV cache usage: 0.0%
INFO 05-17 08:13:05 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.7 tokens/s, Running: 18 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 15.4%, CPU KV cache usage: 0.0%
INFO:     ::1:38684 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:13:11 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.8 tokens/s, Running: 17 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38648 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:13:16 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 13.6 tokens/s, Running: 16 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.2%, CPU KV cache usage: 0.0%
INFO 05-17 08:13:22 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 13.7 tokens/s, Running: 16 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 14.4%, CPU KV cache usage: 0.0%
INFO:     ::1:38850 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:13:28 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 13.3 tokens/s, Running: 15 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 13.6%, CPU KV cache usage: 0.0%
INFO 05-17 08:13:34 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.9 tokens/s, Running: 15 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 13.8%, CPU KV cache usage: 0.0%
INFO:     ::1:38606 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38748 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:13:39 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.7 tokens/s, Running: 13 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 12.2%, CPU KV cache usage: 0.0%
INFO 05-17 08:13:45 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.6 tokens/s, Running: 13 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 12.4%, CPU KV cache usage: 0.0%
INFO:     ::1:38754 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38724 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:13:50 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.9 tokens/s, Running: 11 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 10.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38778 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:13:56 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.0 tokens/s, Running: 10 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 9.9%, CPU KV cache usage: 0.0%
INFO 05-17 08:14:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.0 tokens/s, Running: 10 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 10.0%, CPU KV cache usage: 0.0%
INFO 05-17 08:14:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.0 tokens/s, Running: 10 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 10.2%, CPU KV cache usage: 0.0%
INFO 05-17 08:14:12 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.0 tokens/s, Running: 10 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 10.3%, CPU KV cache usage: 0.0%
INFO:     ::1:38856 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:14:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 9.3 tokens/s, Running: 9 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 9.5%, CPU KV cache usage: 0.0%
INFO 05-17 08:14:23 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.0 tokens/s, Running: 9 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 9.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38722 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:14:28 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.0 tokens/s, Running: 8 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 8.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38814 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:14:33 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.4 tokens/s, Running: 7 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 7.7%, CPU KV cache usage: 0.0%
INFO 05-17 08:14:38 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.9 tokens/s, Running: 7 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 7.9%, CPU KV cache usage: 0.0%
INFO 05-17 08:14:44 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.9 tokens/s, Running: 7 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 8.0%, CPU KV cache usage: 0.0%
INFO:     ::1:38870 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:38712 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:14:49 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 9.7 tokens/s, Running: 5 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 5.9%, CPU KV cache usage: 0.0%
INFO:     ::1:38860 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:14:54 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 8.8 tokens/s, Running: 4 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.8%, CPU KV cache usage: 0.0%
INFO:     ::1:38676 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:14:59 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 7.8 tokens/s, Running: 3 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.7%, CPU KV cache usage: 0.0%
INFO:     ::1:38824 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:15:04 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 7.9 tokens/s, Running: 2 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.6%, CPU KV cache usage: 0.0%
INFO:     ::1:38842 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 08:15:09 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 6.6 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 1.4%, CPU KV cache usage: 0.0%
INFO 05-17 08:15:15 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 1.5%, CPU KV cache usage: 0.0%
INFO:     ::1:38642 - "POST /v1/completions HTTP/1.1" 200 OK
Results written to .artifact/benchmarks/results/aff63235-06a6-4e0f-99ee-97a3c5a23014.jsonl
Killing process 90586...
/var/spool/slurmd/job3676358/slurm_script: line 45: 90554 Killed                  apptainer run --nv --home /mnt/petrelfs/huqinghao/xzyao/:/home/xiayao --bind /mnt/petrelfs/huqinghao/xzyao/code/vllm:/vllm --bind /mnt/petrelfs/huqinghao/xzyao/code/triteia:/triteia --env PYTHONPATH=/vllm:/triteia --env CUDA_LAUNCH_BLOCKING=1 --env TVM_TARGET=nvidia/nvidia-a100 --env TRANSFORMERS_OFFLINE=1 --env NUMEXPR_MAX_THREADS=128 --env USE_BITBLAS=1 --env BITBLAS_TARGET=nvidia/nvidia-a100 --env RAY_DEDUP_LOGS=0 --workdir /vllm /mnt/petrelfs/huqinghao/xzyao/images/vllm-0.4.1.sif python3 /vllm/vllm/entrypoints/openai/api_server.py --model /vllm/.idea/full_models/Llama-2-13b-hf --enable-delta --disable-log-requests --gpu-memory-utilization 0.85 --delta-modules delta-1=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-1 delta-2=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-2 delta-3=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-3 delta-4=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-4 delta-5=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-5 delta-6=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-6 delta-7=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-7 delta-8=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-8 delta-9=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-9 delta-10=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-10 delta-11=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-11 delta-12=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-12 delta-13=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-13 delta-14=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-14 delta-15=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-15 delta-16=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-16 delta-17=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-17 delta-18=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-18 delta-19=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-19 delta-20=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-20 delta-21=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-21 delta-22=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-22 delta-23=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-23 delta-24=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-24 delta-25=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-25 delta-26=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-26 delta-27=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-27 delta-28=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-28 delta-29=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-29 delta-30=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-30 delta-31=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-31 delta-32=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-32 --max-deltas 32 --tensor-parallel-size 4 --enforce-eager
